src/main/java/com/passbook/sparkeighteen/service/ProfileService.java

```diff
@@ -0,0 +1,34 @@
+package com.passbook.sparkeighteen.service;
+
+import com.passbook.sparkeighteen.peristence.POJO.ProfileRequest;
+import com.passbook.sparkeighteen.peristence.POJO.ProfileResponse;
+import com.passbook.sparkeighteen.peristence.entity.ProfileEntity;
+import com.passbook.sparkeighteen.peristence.repository.ProfileRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProfileService {
```

Comment:
Why do you need a separate service for profile? Already user service is there! Profile belongs to a user, code has to be there in user service.